README.md

```diff
@@ -117,7 +117,13 @@ Description of `config.yml` attributes
 
 *Note*: Paths can be specified relative to the `config.yml` file or as absolute paths.
 
-6. Obtain Google credential Files
+6. Login to Synapse by using the command line
+On the CLI in your virtual environment, run the following command: 
+`synapse login -u <synapse username> -p <synapse password> --rememberMe`
```

Comment:
Can you switch to triple block quotes to match the formatting of the other commands in this section?